Makefile

```diff
@@ -1,5 +1,5 @@
 PACKAGE_NAME=github.com/projectcalico/calicoctl
-GO_BUILD_VER=v0.49
+GO_BUILD_VER=v0.72.1
```

Comment:
Let us revisit later if we can upgrade to a more recent GO_BUILD_VER like v0.82, golang 1.19 or 1.20 and k8s.io v1.20.15 APIs.

Reply:
yeah it was done, but everything breaks when we do that. Please look at the commit history.